people.json

```diff
@@ -4,5 +4,11 @@
     "github": "@RamonGebben",
     "twitter": "@RamonGebben",
     "linkedin": "https: //www.linkedin.com/in/ramon-gebben-88039b85"
+  },
+  {
+    "name": "Vaibhav Tandon",
+    "github": "@vaibhav111tandon",
+    "twitter": "NA",
```

Comment:
`"twitter": "NA",` should be `"twitter": "N/A",`

Reply:
Ok

Reply:
Done